Microsoft.HealthVault.Fhir/Transformers/ObservationToHeartRate.cs

```diff
@@ -25,7 +25,11 @@ internal static HeartRate ToHeartRate(this Observation observation)
             }
             
             heartRate.Value = (int)observationValue.Value.Value;
-            heartRate.When = ObservationToHealthVault.GetHealthVaultTimeFromEffectiveDate(observation.Effective);
+
+            if (observation.Effective == null)
```

Comment:
Close/open brackets. Also, if we are throwing an exception a more descriptive exception is better: "Effective field with a valid Effectiva date on observation is required". Although I am not sure why we should throw an exception here, what made you modify this?